aws/resource_aws_ecs_task_definition.go

```diff
@@ -413,6 +476,28 @@ func flattenPlacementConstraints(pcs []*ecs.TaskDefinitionPlacementConstraint) [
 	return results
 }
 
+func flattenProxyConfiguration(pc *ecs.ProxyConfiguration) []map[string]interface{} {
+	if pc == nil {
+		return nil
+	}
+
+	meshProperties := make(map[string]string)
+	if pc.Properties != nil {
+		for _, prop := range pc.Properties {
+			meshProperties[*prop.Name] = *prop.Value
```

Comment:
To prevent potential panics, we should use the AWS Go SDK provided conversion functions here and below:

```suggestion
			meshProperties[aws.StringValue(prop.Name)] = aws.StringValue(prop.Value)
```